CONTRIBUTING-development.md

```diff
@@ -0,0 +1,149 @@
+# Coding style
+
+* We almost always use **CamelCase**, when naming functions, variables etc., **not snake_case**.
+
+* We also have the habits of using a **lowerCamelCase** for some variables named from their type or from their function name, as shown in the examples:
+
+  ```c
+  AlterTableCmd *alterTableCommand = NULL;
+  
+  ..
+  ..
+  
+  bool
+  IsAlterTableRenameStmt(RenameStmt *renameStmt)
+  {
+    bool isAlterTableRenameStmt = false;
+    ..
+  }
+
```

Comment:
```suggestion
```
```

Reply:
what this is missing now?

Reply:
If I'm correct a codeblock is open on line 7 but is not closed on line 19 where it probably should (line 20 being a new bullet point)